packages/api/src/controllers/customAnimalBreedController.js

```diff
@@ -0,0 +1,44 @@
+/*
+ *  Copyright 2024 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+import CustomAnimalBreedModel from '../models/customAnimalBreedModel.js';
+
+const customAnimalBreedController = {
+  getCustomAnimalBreeds() {
+    return async (req, res) => {
+      try {
+        const { farm_id } = req.headers;
+        const { default_type_id, custom_type_id } = req.query;
```

Comment:
Should this check that at least 1 is null?

Reply:
Yup, good call!